app/src/main/java/cm/aptoide/pt/promotions/ClaimPromotionDialogFragment.java

```diff
@@ -0,0 +1,406 @@
+package cm.aptoide.pt.promotions;
+
+import android.content.ClipData;
+import android.content.ClipboardManager;
+import android.content.Context;
+import android.graphics.Point;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.DialogFragment;
```

Comment:
I used the V4 because DialogFragment exists on the V4 package. I did not however consider the V7 version, we can look into it for the future DialogFragment refactor but for now I wouldn't touch it as it is working;